cantonesedetect/judge.py

```diff
@@ -0,0 +1,274 @@
+import re, math
+from typing import List, Tuple, Dict
+
+# Cantonese characters not found in SWC
+CANTO_FEATURE_RE = re.compile(
+    r'[嘅嗰啲咗佢喺咁噉冇啩哋畀嚟諗惗乜嘢閪撚𨳍𨳊瞓睇㗎餸𨋢摷喎嚿噃嚡嘥嗮啱揾搵喐逳噏𢳂岋糴揈捹撳㩒𥄫攰癐冚孻冧𡃁嚫跣𨃩瀡氹嬲掟孭黐唞㪗埞忟𢛴嗱係唔喇俾]|'
+    r'唔[係得會好識使洗駛通知到去走掂該錯差]|點[樣會做得解]|[琴尋噚聽第]日|[而依]家|家[下陣]|[真就實梗又話都]係|邊[度個位科]|'
+    r'[嚇凍攝整揩逢淥浸激][親嚫]|[橫搞傾諗得唔]掂|仲[有係話要得好衰唔]|返[學工去歸]|執[好生實返輸]|[留坐剩]低|'
+    r'屋企|收皮|慳錢|傾[偈計]|幫襯|求其|是[但旦]|[濕溼]碎|零舍|肉[赤緊酸]|核突|同埋|勁[秋抽]|邊[度隻條張樣個]|去邊'
+)
+
+# A list of exceptions where the above characters can be found in SWC
+CANTO_EXCLUDE_RE = re.compile(r'(關係|吱唔|咿唔|喇嘛|喇叭|俾路支|俾斯麥)')
+
+# SWC characters that are less common in Cantonese
+SWC_FEATURE_RE = re.compile(r'[這哪唄咱啥甭那是的他她它吧沒麼么些了卻説說吃弄把也在]|而已')
+
+# A list of exceptions where the above characters can be found in Cantonese (mainly phrases or proper nouns)
+SWC_EXCLUDE_RE = re.compile(
+    r'亞利桑那|剎那|巴塞羅那|薩那|沙那|哈瓦那|印第安那|那不勒斯|支那|'
+    r'是[否日次非但旦]|[利於]是|唯命是從|頭頭是道|似是而非|自以為是|俯拾皆是|撩是鬥非|莫衷一是|唯才是用|'
+    r'[目綠藍紅中]的|的[士確式]|波羅的海|眾矢之的|的而且確|大眼的度|的起心肝'
+    r'些[微少許小]|'
+    r'[淹沉浸覆湮埋沒出]沒|沒[落頂收]|神出鬼沒|'
+    r'了[結無斷當然哥結得解事之]|[未明]了|不得了|大不了|'
+    r'他[信人國日殺鄉]|[其利無排維結]他|馬耳他|他加祿|他山之石|'
+    r'其[它]|'
+    r'[酒網水貼]吧|吧[台臺枱檯]|'
+    r'[退忘阻]卻|卻步|'
+    r'[遊游小傳解學假淺眾衆訴論][説說]|[說説][話服明]|自圓其[説說]|長話短[說説]|不由分[說説]|'
+    r'吃[虧苦力]|'
+    r'弄[堂]'
+    r'把[握柄持火風關鬼口嘴戲脈炮砲屁手聲]|大把|拉把|冧把|掃把|拖把|得把|加把|下把位|一把年紀|把死人聲|自把自為|兩把|三把|四把|五把|幾把|拎把|第一把|泵把|'
+    r'也[許門]|[非威]也|也文也武|之乎者也|維也納|空空如也|頭也不回|時也[命運]也|'
+    r'在[場乎下校學行任野意於望內案旁生世心線逃位即職座囚此家]|[站志旨爭所勝衰實內外念現好健存潛差弊活]在|我思故我在'
+)
+
+# A list of quotes: Content inside and outside a pair of quotes should be treated separately.
+ALL_QUOTEMARKS_RE = re.compile(r'「[^「]*」|“[^“]*”|《[^《]*》|【[^【]*】|『[^『]*』')
+
+# A list of sentential delimiters
+ALL_DELIMITERS_RE = re.compile(r'[，。；？！⋯\n]')
```

Comment:
`⋯` is defined as a mathematics symbol, use `…` instead
(I think we should use the `regex` module which supports Unicode categories though)